gulpfile.js

```diff
@@ -89,9 +89,13 @@ var opts = {
   },
   distPreprocess: ['generate-index-icons', 'dist-css'],
   scsslint: true,
-  testPaths: [
-    'test/**/*.js'
-  ]
+
+  test: {
+    paths: ['test/**/*.js'],
+    // You may pass in a path or a custom config obj
+    pathToBabelRc: __dirname+'/.babelrc'
```

Comment:
I'm wondering if there is a way to automatically identify this without exposing the path to babel.

If you move this to be inside ` src/utils/gulp/gulp-tasks-test.js` would that work?

Ultimately, I believe we could achieve this:
  1 - Try to load `require('fs').readFileSync(path.join(__dirname, '.babelrc'), 'utf8')`
  2 - If root .babelrc is not loaded, use the default one.